django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,13 @@
+import enum
+
 from django.db import router
 
 
+class SeverityType(enum.Enum):
```

Comment:
Based on the evolution of this issue, I think **category** is a better word instead of **severity**.
Thoughts?

Reply:
`OperationType` or `OperationCategory` makes sense to me, as they are showing operations.